ubuntu1604/cloudbuild.yaml

```diff
@@ -37,7 +37,7 @@ steps:
 # We use container_test rule, which is a Bazel wrapper of container_structure_test.
 # https://github.com/bazelbuild/rules_docker/blob/master/contrib/test.bzl
   - name: "l.gcr.io/google/bazel"
-    args: ["--output_base=/workspace/ubuntu1604", "test", "//:image-test"]
+    args: ["--output_base=/workspace/ubuntu1604", "test", "--test_output=errors", "//:image-test"]
```

Comment:
And to debian9 and centos7 as well?

Reply:
debian9 & centos7 already have this option